config/samples/config_v1beta1_policy_rego.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: config.ratify.deislabs.io/v1beta1
```

Comment:
suggestion to move new samples into subfolder, as this folder is growing

Reply:
good call.